packages/common/src/types/schema.ts

```diff
@@ -2765,17 +2865,18 @@ export type UseSuggestedQuantityInput = {
 
 export type UseSuggestedQuantityResponse = RequisitionLineConnector | UseSuggestedQuantityError;
 
-export type User = {
-  __typename: 'User';
+export type UserNameDoesNotExist = AuthTokenErrorInterface & {
+  __typename: 'UserNameDoesNotExist';
+  description: Scalars['String'];
+};
+
+export type UserNode = {
+  __typename: 'UserNode';
   /** The user's email address */
   email?: Maybe<Scalars['String']>;
   /** Internal user id */
   userId: Scalars['String'];
-};
-
-export type UserNameDoesNotExist = AuthTokenErrorInterface & {
-  __typename: 'UserNameDoesNotExist';
-  description: Scalars['String'];
+  username: Scalars['String'];
```

Comment:
🤔 

Would be cool to have a little popover when you hover a user name which shows details like their email and name etc